galsim/table.py

```diff
@@ -165,18 +179,22 @@ def __call__(self, x):
             if dimen > 2:
                 raise ValueError("Arrays with dimension larger than 2 not allowed!")
             elif dimen == 2:
-                f = np.zeros_like(x)
-                for i in xrange(x.shape[0]):
-                    f[i,:] = np.fromiter((self.table(float(q)) for q in x[i,:]), dtype='float')
+                f = np.empty_like(x.flatten(), dtype=float)
```

Comment:
Oh.  The reason for my comment above is that when I used grep I found a whole bunch of ```flatten```s, but it turns out I hadn't pulled the latest version.  Sorry.